client/src/components/new-request/analyze-pending.vue

```diff
@@ -1,17 +1,17 @@
 <template>
   <MainContainer id="analyze-pending-container">
     <template v-slot:container-header>
-      <v-col cols="auto h4">
+      <v-col cols="auto font-weight-bold h6 ml-3">
```

Comment:
A bold H6? This smells like it should have its own style.

Reply:
Good idea, will look at this!

Reply:
Oh man, i did this again within the `col`, or did i accidentally add to it. Will fix

Reply:
Fixed 👍